### PR TITLE
linux/sound: Fix HTML/C++ quoting

### PR DESCRIPTION
**Pull request by hoffie**
As spotted by @passing in https://github.com/jamulussoftware/jamulus/pull/1127#discussion_r606487013.
    
Also update translation texts to avoid unnecessary work by translators.

### CODE REVIEW

**Review by pljones — APPROVED** [review]

File: linux/sound.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ void CSound::OpenJack ( const bool  bNoAutoJackConnect,
         throw CGenErr ( tr ( "The Jack server sample rate is different from "
             "the required one. The required sample rate is:" ) + " <b>" +
             QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz</b>. " + tr ( "You can "
-            "use a tool like <i><a href=""http://qjackctl.sourceforge.net"">QJackCtl</a></i> "
+            "use a tool like <i><a href=\"http://qjackctl.sourceforge.net\">QJackCtl</a></i> "
```
Comment (ann0see) [inline linux/sound.cpp:63]:
```suggestion
            "use a tool like <i><a href=\"https://qjackctl.sourceforge.io\">QJackCtl</a></i> "
```

They're using https now.

File: src/res/translation/translation_de_DE.ts [hunk 2]
```diff
@@ -3299,8 +3299,8 @@
     </message>
     <message>
         <location filename="../../../linux/sound.cpp" line="62"/>
-        <source>You can use a tool like &lt;i&gt;&lt;a href=http://qjackctl.sourceforge.net&gt;QJackCtl&lt;/a&gt;&lt;/i&gt; to adjust the Jack server sample rate.</source>
-        <translation>Do kannst ein Werkzeug wie &lt;i&gt;&lt;a href=http://qjackctl.sourceforge.net&gt;QJackCtl&lt;/a&gt;&lt;/li&gt; verwenden um die Samplerate umzustellen.</translation>
+        <source>You can use a tool like &lt;i&gt;&lt;a href=&quot;http://qjackctl.sourceforge.net&quot;&gt;QJackCtl&lt;/a&gt;&lt;/i&gt; to adjust the Jack server sample rate.</source>
+        <translation>Do kannst ein Werkzeug wie &lt;i&gt;&lt;a href=&quot;http://qjackctl.sourceforge.net&quot;&gt;QJackCtl&lt;/a&gt;&lt;/li&gt; verwenden um die Samplerate umzustellen.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_de_DE.ts:3303]:
```suggestion
        <translation>Du kannst ein Werkzeug wie &lt;i&gt;&lt;a href=&quot;http://qjackctl.sourceforge.net&quot;&gt;QJackCtl&lt;/a&gt;&lt;/li&gt; verwenden um die Samplerate umzustellen.</translation>
```

**Review by ann0see — CHANGES_REQUESTED** [review]
Thanks! Please see my comments.

**Comment by hoffie** [comment]
@ann0see Thanks for the hints. I fixed this in separate commits as the fixes are unrelated to the original issue. Therefore, please don't squash-merge. :)

**Comment by ann0see** [comment]
Argh. Sorry I didn't make it clear enough: it's https://\*.sourceforge.**io** if you use https not https://\*.sourceforge.**net**

**Comment by hoffie** [comment]
> Argh. Sorry I didn't make it clear enough: it's https://*.sourceforge.**io** if you use https not https://*.sourceforge.**net**

Ah, I had missed this. Should have checked more thoroughly. Should be fixed now.

**Review by ann0see — APPROVED** [review]
Great

**Comment by ann0see** [comment]
Thanks!

**Pull request merged**